docs/extensions/IconMarkupExtensions.md

```diff
@@ -0,0 +1,126 @@
+---
+title: FontIcon, FontIconSource, BitmapIcon markup extensions
+author: sergio0694
+description: The FontIcon, FontIconSource and BitmapIcon markup extensions allow developers to easily declare these types of icons directly from XAML in a compact manner.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, nullable bool, dependency property, markup extension, XAML, markup 
+---
+
+## IconMarkupExtensions
```

Comment:
@Sergio0694 the headings here being smaller and then larger throws off the doc preview.

Can we adjust this file so this is the main '#' header, and we use '##' for the name of each extension, and then '###' for the Syntax/Properties?

I think that'll mean we'll lose the Syntax/Properties from the right-hand navigation, but then the name of each extension will be visible there instead.

Reply:
Done in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/313/commits/2fcffe906c22a391b0caf8febbe8320978d17586, is this alright?

Reply:
Will check the preview in a couple min, but yup! I added too more comments below, sorry for the confusion!